supereeg/model.py

```diff
@@ -597,7 +605,7 @@ def _create_locs(self, locs, template):
     if self.locs.shape[0]>1000:
         warnings.warn('Model locations exceed 1000, this may take a while. Go get a cup of coffee or brew some tea!')
 
-def _bo2model(bo, locs, width=20):
+def _bo2model(bo, locs, width=20, disable_parallelization=False):
```

Comment:
do we still need the `disable_parallelization` flag?

Reply:
nope, removed